website/server/controllers/api-v3/tasks.js

```diff
@@ -542,10 +542,37 @@ api.scoreTask = {
 
     let wasCompleted = task.completed;
 
+    let previousStats = _.clone(user.stats);
```

Comment:
clone doesn't work well on mongoose objects (especially in v4), it's better to call _.clone(user.stats.toObject())